auth/auth.css

```diff
@@ -0,0 +1,56 @@
+@import url("/index.css");
+
+.input-container{
+    max-width: 500px;
+    margin:25vh auto;
```

Comment:
you can use rem here

Reply:
Did the changes. Thank you for reviewing and approving.